### ISSUE TITLE
Assets-related crashes after update on Android

### ISSUE THREAD

**Issue by mhagmajer** [Microsoft/react-native-code-push#183]
After updating `react-native` to `0.19.0` and `react-native-code-push` to `1.7.1-beta`, any code pushes to my app caused it to crash. Java exceptions were related to the app not being able to find the bundled resources. I followed the instructions carefully making sure that the JS bundle file is named correctly and that all the assets were bundled along and that code-push sent them to the app just fine. This crash also made the app unable to receive any additional updates...

I solved the problem by moving all static image resources to Android drawable resources and publishing a new version in the Play store. I'm not able to code-push any additional assets though which is a bummer...

@lostintangent I think this is related to the recent support for bundling assets for Android. How can we debug this more?

**Comment by geof90** [comment]
Could we see the exception thrown? That could be helpful!

**Comment by geof90** [comment]
Also, how did you update to react-native 0.19.0? did you use the react-native upgrade command? My quick guess could be that Gradle is still sync-ing an older version of the react-native native libraries.

**Comment by lostintangent** [comment]
Hey @mhagmajer! I'm super sorry to hear this! We haven't seen issues like this before, so I'd love to get to the bottom of it and figure out how we can either fix a bug in the product, make sure the docs are clearer, etc. Is this the error message that you saw? 

`Update is invalid - no files with extension .bundle, .js or .jsbundle were found in the update package.`

Additionally, could you share an example of the `react-native bundle` and `code-push release` commands you ran? Just to help us double-check that we know the workflow you used.

**Comment by mhagmajer** [comment]
Guys, and especially @lostintangent, thank you for very informative responses! :+1:

I wish I could present you with more details on the issue I encountered, however, I couldn't repro the problem after I moved the resources to within the app. I also checked if the assets updates work in a fresh project on RN 0.19.0 and RNCP 1.7.1-beta and they do. This is not a pressing matter to the development of our app and I can't investigate more what happened there right now. Next time we need to ship resources as an update, I'll keep a close look at this, and will reopen the issue with as much details as I can.

I'd like to take this opportunity to say how much I appreciate your work on this. Ability to immediately push updates to native client has fasten our development a lot. Keep it up! :)
